src/main/java/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,24 @@
+package strategy;
+
+import handler.OperationHandler;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<FruitTransaction.Operation, OperationHandler> handlers;
```

Comment:
Consider encapsulating the handlers map by making it private and providing no direct access to it. This will adhere to the encapsulation principle.